### DISCUSSION TITLE
Welcome to Termux Plus Discussions!

### DISCUSSION THREAD

**Discussion by leap0x7b** [leapofazzam123/termux-plus#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by creepy-pasta101** [comment]
@leapofazzam123 I have other java packages in my repositories tho ... but those are huge... about 200 MB If you can add it.. you can do so

**Reply by leap0x7b** [reply]
Sure, in your java-in-termux repo, only ARM is available for JDK8 and JDK11. Can you make debs for JDK8 and JDK11 for multiple architecture?

**Reply by creepy-pasta101** [reply]
Nope... I am trying still tho.... those were precompiled binaries when I got them...
I can try Java11 but JDK8 is by @Hax4us ...

**Reply by leap0x7b** [reply]
ok, but the hax4us one isn't in deb, can you repack it to a deb

**Reply by creepy-pasta101** [reply]
yeah sure...

**Reply by creepy-pasta101** [reply]
Also... those java debs cannot be hosted directly tho... github only allows files =< 100 MB to be hosted on source code repo... rest will be redirected to git-lfs.. which creates problems...

**Comment by leap0x7b** [comment]
Also looks like mjuned47 only compiles libspice and QEMU with SPICE for ARM architectures. Can you compile libspice and QEMU with SPICE for multiple architectures with patches from this https://github.com/mjuned47/qemu-termux/tree/master/patches

**Reply by creepy-pasta101** [reply]
Sure... will do

**Reply by creepy-pasta101** [reply]
Seems like spice-server cannot be compiled that easily by that single patch.....
It requires more patches...
Meanwhile I could compile ORC and spice-protocol

**Comment by creepy-pasta101** [comment]
I was able to compile spice-server....
Will publish soon at Tewmux-disabled
